_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -149,6 +149,22 @@ Option | Required | Type   | Description
 `header` | No   | String list | The header containing the column names used to parse CSV data.
 `detect_header` | No   | Boolean   | Whether the first line of the S3 object should be interpreted as a header. Default is `true`.
 
+## Codec and processor combinations 
+
+At ingestion time, data received by the S3 source can be parsed by codecs. However, codecs only support a particular type of data input type, compressions types, and processors. The following table describes each codec, data type, and processor combination where performance of the pipeline can be improved. 
+
+| **Input Data Type** | **Compression type** | **Codec** | **Processor** | Notes |
```

Comment:
Rather than adding a table here, perhaps we can link to the other page for users.

Reply:
I'm going to remove the table.